docs/_includes/generated/resource_manager_configuration.html

```diff
@@ -0,0 +1,36 @@
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 65%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>containerized.heap-cutoff-min</h5></td>
+            <td>600</td>
+            <td>Minimum amount of heap memory to remove in containers, as a safety margin.</td>
+        </tr>
+        <tr>
+            <td><h5>containerized.heap-cutoff-ratio</h5></td>
+            <td>0.25</td>
+            <td>Percentage of heap space to remove from containers (YARN / Mesos), to compensate for other JVM memory usage.</td>
+        </tr>
+        <tr>
+            <td><h5>local.number-resourcemanager</h5></td>
+            <td>1</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><h5>resourcemanager.job.timeout</h5></td>
+            <td>"5 minutes"</td>
+            <td>Timeout for jobs which don't have a job manager as leader assigned.</td>
+        </tr>
+        <tr>
+            <td><h5>resourcemanager.rpc.port</h5></td>
```

Comment:
Why does this not have the description from the original doc?

Reply:
Must've missed it by accident, added it back.